lib/benchee/system.ex

```diff
@@ -29,9 +29,9 @@ defmodule Benchee.System do
     %Suite{suite | system: system_info}
   end
 
-  defp elixir, do: System.version()
-
-  defp erlang do
+  @doc false
+  @spec erlang() :: String.t()
+  def erlang do
```

Comment:
why did you make this public? Can't seem to find a call?

Reply:
I _had_ used that previously when I was using the Erlang version to determine the reduction count offset, but when I moved over to calculating it the right way I forgot to make it private again. Pushed up that change now!